Program/Sound

```diff
@@ -0,0 +1,121 @@
+import random
+
+frq = 12500
+amp = 10
+chan = 1
+hz = 22050
+numsamples = 10
```

Comment:
[Variable Name Conventions](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)